README.md

```diff
@@ -1,88 +1,88 @@
 # Project
 
-This extension adds a `project` object to the `planning` object.
+This extension adds a `project` object to the `planning` object to describe the infrastructure or Public-Private Partnership (PPP) project to which the contracting process is related.
 
-In OCDS, project information is nested under the [`planning.budget`](https://standard.open-contracting.org/latest/en/schema/reference/#budget) object. However, in some cases, budget management systems and project management systems are separate, and it might be important to separately specify:
+The identifier for the infrastructure or PPP project to which a contracting process is related ought to be disclosed using the `planning.project.id` field.
 
-* The amount reserved in the budget for a specific contracting process
-* The project the contract relates to, and the total value of that project
-* Sector classifications
-* Additional classifications
-* Project locations, with options for gazetteer or point locations
-
-This is particularly important in cases of Public-Private Partnerships and large infrastructure projects, where users might want to track all the contracting processes related to the large-scale project, and to understand the individual contracts in the context of their contracting process and overall project values.
+The `planning.budget.projectID` field ought not be used to disclose the identifier for an infrastructure or PPP project. This field is used to disclose the identifier for a project in the national budget to which the contracting process is related. Since projects in the national budget might include many individual infrastructure projects, it is necessary to disclose these identifiers separately.
 
 This extension must be used with the [Location](https://extensions.open-contracting.org/en/extensions/location/master/) extension.
 
-## Examples
+## Example
+
+A buyer plans a contracting process for the design of a bridge.
+
+The contracting process is part of an infrastructure project which covers the design, construction and supervision of the bridge. Information about the infrastructure project is disclosed in `planning.project`.
+
+The buyer publishes a separate [Open Contracting for Infrastructure Data Standard](https://standard.open-contracting.org/infrastructure/) (OC4IDS) dataset, describing its infrastructure projects. `planning.project.id` and `planning.project.uri` reference the project's identifier and URI in the OC4IDS dataset. `sector` reference's the project's classification in the [OC4IDS sector codelist](https://standard.open-contracting.org/infrastructure/latest/en/reference/codelists/#projectsector).
```

Comment:
```suggestion
The buyer publishes a separate [Open Contracting for Infrastructure Data Standard](https://standard.open-contracting.org/infrastructure/) (OC4IDS) dataset, describing its infrastructure projects. `planning.project.id` and `planning.project.uri` reference the project's identifier and URI in the OC4IDS dataset. `planning.project.sector` reference's the project's classification in the [OC4IDS sector codelist](https://standard.open-contracting.org/infrastructure/latest/en/reference/codelists/#projectsector).
```

just because you specified `planning.project.` in the other two fields referenced here

Reply:
https://github.com/open-contracting-extensions/ocds_project_extension/pull/29/commits/8dcdfe3a6100e8392ed6071dc4e4e3481da474c1